<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>908</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/images/terminal.png</normaloff>:/icons/images/terminal.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QTableWidget" name="tableWidget_shortcuts"/>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_settings">
      <property name="title">
       <string>Settings</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QCheckBox" name="checkBox_autoclear">
           <property name="text">
            <string>Auto clear output, maximum of characters: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_autoclear_maxCharCnt">
           <property name="maximum">
            <number>100000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_timeLog">
         <property name="text">
          <string>Enable time log</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="checkBox_outputSave">
           <property name="text">
            <string>Save terminal output to:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_save"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_save">
           <property name="text">
            <string>browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QCheckBox" name="checkBox_prefix">
           <property name="text">
            <string>Enable Tx Prefix:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_prefix"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QCheckBox" name="checkBox_suffix">
           <property name="text">
            <string>Enable Tx Suffix:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_suffix"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_clearOutputLine">
         <property name="text">
          <string>Clear output line automatically</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_ascii">
       <attribute name="title">
        <string>ASCII</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTextEdit" name="textEdit_out_ascii"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="lineEdit_find_ascii"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_fnd_ascii">
                <property name="text">
                 <string>Find</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="lineEdit_in_ascii"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_hex">
       <attribute name="title">
        <string>HEX</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTextEdit" name="textEdit_out_hex"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLineEdit" name="lineEdit_find_hex"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_fnd_hex">
                <property name="text">
                 <string>Find</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLineEdit" name="lineEdit_in_hex"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_dec">
       <attribute name="title">
        <string>DEC</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QTextEdit" name="textEdit_out_dec"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLineEdit" name="lineEdit_find_dec"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_fnd_dec">
              <property name="text">
               <string>Find</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLineEdit" name="lineEdit_in_dec"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_script">
      <item>
       <widget class="QLabel" name="label_script">
        <property name="text">
         <string>Script:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_script"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_script_dataType"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_script_repeat">
        <property name="text">
         <string>repeat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_script_period">
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_script_run">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
